packages/cli/src/commands/setup/graphql/graphql.js

```diff
@@ -0,0 +1,14 @@
+import terminalLink from 'terminal-link'
+
+export const command = 'graphql <feature>'
+export const description = 'Set up GraphQL feature support'
+export const builder = (yargs) =>
+  yargs
+    .commandDir('./features')
```

Comment:
@jtoar was trying to get rid of our usage of `commandDir` (see https://github.com/redwoodjs/redwood/pull/9750), but ran into some non-trivial issues. Let's not add any new code that uses `commandDir`, to not make it even more difficult to get rid of in the future.

Reply:
Done.